docs/source/user_guide/model_understanding.ipynb

```diff
@@ -763,6 +792,36 @@
     "\n",
     "The returned `objective_thresholds` dictionary has the objective measure as the key, and the dictionary value associated contains both the best objective score and the threshold that results in the associated score."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualize high dimensional data in lower space\n",
+    "\n",
+    "We can use T-SNE to visualize data with many features on a 2D plot, making it easier to see relationships in your data."
```

Comment:
Might be nice to include a link to the documentation for this function

Reply:
I think just a link to [this](https://evalml.alteryx.com/en/stable/autoapi/evalml/model_understanding/index.html#evalml.model_understanding.graph_t_sne) would help!